lang/java/ipc/src/test/java/org/apache/avro/DataFileInteropTest.java

```diff
@@ -40,27 +40,27 @@ public static void printDir() {
         + DATAFILE_DIR.getAbsolutePath());
   }
 
-  @Test
-  public void testGeneratedGeneric() throws IOException {
-    System.out.println("Reading with generic:");
-    DatumReaderProvider<Object> provider = new DatumReaderProvider<Object>() {
-      @Override public DatumReader<Object> get() {
-        return new GenericDatumReader<>();
-        }
-      };
-    readFiles(provider);
-  }
-
-  @Test
-  public void testGeneratedSpecific() throws IOException {
-    System.out.println("Reading with specific:");
-    DatumReaderProvider<Interop> provider = new DatumReaderProvider<Interop>() {
-      @Override public DatumReader<Interop> get() {
-        return new SpecificDatumReader<>();
-        }
-      };
-    readFiles(provider);
-  }
+  // @Test
```

Comment:
Why are these commented out?  Are they failing?  If they're no longer required then we should just delete them.